acceptance_tests/features/edit_survey_details.feature

```diff
@@ -1,17 +1,16 @@
+@standalone
 Feature: As an internal user
   I need to be able to edit/amend survey details
   So that I can change/update survey details when required
 
   Background: Internal user is already signed in
   Given the internal user is already signed in
 
+  @fixture.setup.data.create_survey
```

Comment:
rename to something like @fixture.setup.data.with.internal.user + associated method - after survey created, calling create_response_user() (feel free to rename to create_internal_user()), gives the scenerio it's own internal user login account